build_all.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+
+#./docker-builder/build_project.sh $PATH_TO_SOURCE_CODE "↓↓↓ build command ↓↓↓" 
+
+#echo "===Building for Maya 2020..."
+#rm -rf ./excons.cache
+#scons with-maya=2020 devtoolset=11
```

Comment:
If we need this code in the future, we can restore it from history.
Right now it's not clear why it was commented. If it should be restored in the future, there should be a `# FIXME` or `# TODO` comment explaining it.

For example:
`# TODO Uncomment when Maya 2020 will be installed in docker`

Reply:
update build_all.sh in commit 0377768. For maya2020 is python 2, I deleted the comment "building for maya2020 ".